locales/en.yml

```
@@ -283,3 +283,16 @@ TInputProfile:
   addInstagram: (add from Instagram)
   addFacebook: (add from Facebook)
   placeholder: Search on WeDance, Instagram or Facebook
+
+EventDashboard:
+  main: Only event owner can access this area.
```

Give more context to translators. It's not "main" section. It's just an error message that user don't have an access.

```suggestion
  noAccess: Only event owner can access this area.
```